src/test/versionbits_tests.cpp

```diff
@@ -429,7 +429,7 @@ BOOST_AUTO_TEST_CASE(versionbits_computeblockversion)
             const uint32_t dep_mask{g_versionbitscache.Mask(chainParams->GetConsensus(), dep)};
             BOOST_CHECK(!(chain_all_vbits & dep_mask));
             chain_all_vbits |= dep_mask;
-            check_computeblockversion(chainParams->GetConsensus(), dep);
+            check_computeblockversion(g_versionbitscache, chainParams->GetConsensus(), dep);
```

Comment:
eca22c726ac48b4216bb68cc0f0bbd655c43ac12 nit: Later diff could be smaller by a few lines if you added a `VersionBitsCache& vbcache{g_versionbitscache};` in this commit already.